src/uwtools/tests/cli/test_templater.py

```diff
@@ -220,12 +222,11 @@ def test_set_template_verbosity(sw, capsys):
             templater.main()
     actual = capsys.readouterr().out.split("\n")
     for line in expected.split("\n"):
-        assert line_in_lines(line, actual)
+        assert any(re.match(r"^.*: %s$" % re.escape(line), x) for x in actual)
```

Comment:
We were running with the intent to avoid regex if at all possible. I may have also started with a significantly more cursed and illegible regex as well - I'm going to try to internalize this for "how to write actually readable regex"!

Reply:
I'll have another PR ready today or tomorrow that negates the need for the regex, too.